Code/BuildSystem/CMake/ezUtilsTargetCS.cmake

```diff
@@ -0,0 +1,61 @@
+######################################
+### ez_create_target_cs(<LIBRARY | APPLICATION> <target-name>)
+######################################
+
+function(ez_create_target_cs TYPE TARGET_NAME)
+
+    set(ARG_OPTIONS NO_PCH NO_UNITY NO_QT NO_EZ_PREFIX)
```

Comment:
Looks like you are not using most of these parameter options

Reply:
Yep, did some cleanup.